### PR TITLE
Add AWS CLI commands for aws ec2 tutorial

### PR DESCRIPTION
**Pull request by Waterdrips**
This adds AWS CLI commands to the AWS EC2 tutorial. These have been
tested by creating the user, policy and attaching them on the CLI.
This has been tested on Linux x86 only, I do not have access to
macosx/windows.

Signed-off-by: Alistair Hey <alistair@heyal.co.uk>

### CODE REVIEW

File: docs/tools/inletsctl.md [hunk 1]
```diff
@@ -252,16 +260,33 @@ Query OK, 1 row affected (0.039 sec)
...
+```
+
 * Add the Policy to the IAM user
-* Generate an access key for your IAM User
 
-Save the access-key as `access-key.txt` and your secret-key as `secret-key.txt`
+We need to use the policy arn generated above, it should have been printed to the console on success. It also follows the format below.
+
+```sh 
+aws iam attach-user-policy --user-name inlets-automation --policy-arn arn:aws:iam::{your-aws-accoutn-number}:policy/inlets-automation
```
Comment (alexellis) [inline docs/tools/inletsctl.md:280]: Typo found: `accoutn`

Can you add:

```
export AWS_ACCOUNT_NUMBER="..."
aws iam attach-user-policy --user-name inlets-automation --policy-arn arn:aws:iam::${AWS_ACCOUNT_NUMBER}:policy/inlets-automation
```

File: docs/tools/inletsctl.md [hunk 2]
```diff
@@ -252,16 +260,33 @@ Query OK, 1 row affected (0.039 sec)
...
+```
+
+* Generate an access key for your IAM User 
+The below commands will create a set of credentials and save them into files for use later on.
+```sh 
+ACCESS_KEY_JSON=$(aws iam create-access-key --user-name inlets-automation)
+echo $ACCESS_KEY_JSON | jq -r .AccessKey.AccessKeyId > access-key.txt
```
Comment (alexellis) [inline docs/tools/inletsctl.md:287]: Link to jq for installation. It's not a standard utility

**Review by alexellis — CHANGES_REQUESTED** [review]
A couple of changes are required please.

File: docs/tools/inletsctl.md [hunk 3]
```diff
@@ -231,15 +231,23 @@ Query OK, 1 row affected (0.039 sec)
 
 ### Example usage with AWS EC2
 
+To use the instructions below you must have the AWS CLI configured with sufficient permissions to 
+create users and roles. 
+
 * Create an IAM user
-* Create a Policy with the following:
+```sh
```
Comment (alexellis) [inline docs/tools/inletsctl.md:238]: Let's create the policy first then the user, then attach.

I do wonder if you can give a policy when creating the user in one step?

Comment (Waterdrips) [reply docs/tools/inletsctl.md]: you can't in 1 step with the cli. Iv moved policy then user

**Review by alexellis — APPROVED** [review]
LGTM

**Pull request merged**